### PR TITLE
tools/tidb-binlog: update faq

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
- Move the FAQ  part in `maintain.md` to `faq.md` to make the docs more organized and focused.

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 
dev, v2.1, v3.0, v3.1

### CODE REVIEW

**Comment by anotherrachel** [comment]
@july2993 @TomShawn  PTAL.

**Comment by TomShawn** [comment]
Would it be better to move the 12 newly added questions and answers after the existing FAQs in this document to make it more progressive and usable?

**Comment by anotherrachel** [comment]
> @anotherrachel Would it be better to move the 12 newly added questions and answers after the existing FAQs in this document to make it more progressive and usable?

I agree. What do you think of this suggestion? @july2993

**Comment by july2993** [comment]
only move ## 同步时出现上游数据库支持但是下游数据库执行会出错的 DDL，应该怎么办？


> > @anotherrachel Would it be better to move the 12 newly added questions and answers after the existing FAQs in this document to make it more progressive and usable?
> 
> I agree. What do you think of this suggestion? @july2993

do you means move all faq in tidb-binlog/faq.md into maintain.md ?
I am ok with this.

**Comment by TomShawn** [comment]
> only move ## 同步时出现上游数据库支持但是下游数据库执行会出错的 DDL，应该怎么办？
> 
> > > @anotherrachel Would it be better to move the 12 newly added questions and answers after the existing FAQs in this document to make it more progressive and usable?
> > 
> > 
> > I agree. What do you think of this suggestion? @july2993
> 
> do you means move all faq in tidb-binlog/faq.md into maintain.md ?
> I am ok with this.

I mean moving the newly added FAQs after the `## 主从同步开启 `ignore-error` 触发 critical error 后如何重新部署` right in this `tidb-binlog/faq.md` document.

**Comment by TomShawn** [comment]
@anotherrachel Please resolve the conflicts.

**Comment by TomShawn** [comment]
@anotherrachel When the content is ready, please also update v3.1 as well, thanks~

**Comment by anotherrachel** [comment]
@july2993 @TomShawn v3.1 also updated, PTAL again. Thanks!

File: dev/reference/tidb-binlog/faq.md [hunk 1]
```diff
@@ -121,3 +121,85 @@ TiDB 配置开启 `ignore-error` 写 binlog 失败后触发 critical error 告
...
+可以使用 update-pump 修改 Pump 状态为 offline 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 update-pump 命令将该 Pump 状态设置为 offline，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 update-pump 将该 Pump 设置为 offline。
+
+在其他情况下一定要使用 offline-pump 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 paused，现在不想使用这个 Pump 了，能否用 update-pump 命令设置状态为 offline？
+
+Pump 以 paused 状态退出进程时，不保证所有 binlog 数据被下游 Drainer 消费。所以这样做会有上下游数据不一致的风险。正确的做法是重新启动 Pump，然后使用 offline-pump 下线该 Pump。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]: Please frame the states and commands with backquotes in your change.

File: dev/reference/tidb-binlog/faq.md [hunk 2]
```diff
@@ -121,3 +121,85 @@ TiDB 配置开启 `ignore-error` 写 binlog 失败后触发 critical error 告
...
+
+## 什么情况下使用 binlogctl 的 update-drainer 命令设置 Drainer 状态为 offline？
+
+- 有从历史任务遗留下来且不再使用的 Drainer 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 update-drainer 将该 Drainer 设置为 offline。
+
+## 可以使用 `change pump`、`change drainer` 等 SQL 操作来暂停或者下线 Pump/Drainer 服务吗？
+
+目前还不支持，这种 SQL 操作会直接修改 PD 中保存的状态，在功能上等同与 binlogctl 的 update-pump、update-drainer 命令。如果需要暂停或者下线，仍然要使用 binlogctl。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
目前还不支持。这种 SQL 操作会直接修改 PD 中保存的状态，在功能上等同于使用 binlogctl 的 update-pump、update-drainer 命令。如果需要暂停或者下线，仍然要使用 binlogctl。
```

File: dev/reference/tidb-binlog/faq.md [hunk 3]
```diff
@@ -121,3 +121,85 @@ TiDB 配置开启 `ignore-error` 写 binlog 失败后触发 critical error 告
...
+
+## 同步时出现上游数据库支持但是下游数据库执行会出错的 DDL，应该怎么办？
+
+1. 查看 drainer.log 日志，查找 `exec failed` 找到 Drainer 退出前最后一条执行失败的 DDL。
+2. 将 DDL 改为下游兼容支持的版本，在下游数据库中手动执行。
+3. 查看 drainer.log 日志，查找执行失败的 DDL 语句，可以查询到该 DDL 的 commit-ts。
+4. 编辑 drainer.toml 配置文件，在 ignore-txn-commit-ts 项中添加该 commit-ts，重启 Drainer。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
4. 编辑 `drainer.toml` 配置文件，在 `ignore-txn-commit-ts` 项中添加该 commit-ts，重启 Drainer。
```

File: dev/reference/tidb-binlog/faq.md [hunk 4]
```diff
@@ -121,3 +121,85 @@ TiDB 配置开启 `ignore-error` 写 binlog 失败后触发 critical error 告
...
+
+在绝大部分情况下，TiDB 和 MySQL 的语句都是兼容的。用户需要注意的是上下游的 sql_mode 应当保持一致。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
在绝大部分情况下，TiDB 和 MySQL 的语句都是兼容的。用户需要注意的是上下游的 `sql_mode` 应当保持一致。
```

File: dev/reference/tidb-binlog/faq.md [hunk 5]
```diff
@@ -115,9 +115,96 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+- 同步任务取消：不再需要将数据同步到某个下游，所以下线对应的 Drainer。
+- 服务器迁移：需要将服务迁移到其他服务器。下线服务，在新的服务器上重新部署。
+
+## 可以通过哪些方式暂停 Pump/Drainer？
+
+- 直接 kill 进程。
+
+    >**注意：**
+    >
+    > 不能使用 kill -9，这种情况 Pump/Drainer 无法对信号进行处理
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
    > 不能使用 `kill -9` 命令，否则 Pump/Drainer 无法对信号进行处理
```

File: dev/reference/tidb-binlog/faq.md [hunk 6]
```diff
@@ -115,9 +115,96 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+
+在某些异常情况下，Drainer 没有正确维护自己的状态，，对数据同步造成了影响，实际上状态应该为 `paused`。这时可以使用 `update-drainer` 对状态进行修正，例如：
+
+- Drainer 异常退出（出现 panic 直接退出进程，或者误操作执行 kill -9 直接 kill 掉进程），Drainer 保存在 PD 中的状态仍然为 `online`。当 Pump 启动时无法正常通知该 Drainer（报错 `notify drainer ...`），导致 Pump 无法正常运行。这个时候可以使用 `update-drainer` 将 Drainer 状态更新为 `paused`，再启动 Pump。
+
+## 可以通过哪些方式下线 Pump/Drainer？
+
+目前只可以使用 binlogctl 的 `offline-pump` 和 `offline-drainer` 命令来下线 Pump 和 Drainer。
+
+## 可以使用 binlogctl 的 `update-pump`/`update-drainer` 命令来下线 Pump/Drainer 服务吗？
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]: Would it be better to move this Q&A to Line 151?

File: dev/reference/tidb-binlog/faq.md [hunk 7]
```diff
@@ -115,9 +115,96 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+> 仅在可以容忍 binlog **数据丢失、上下游数据不一致**或者确认不再需要使用该 Pump 存储的 binlog 数据的情况下，才能使用 `update-pump` 修改 Pump 状态为 `offline`。
+
+可以使用 `update-pump` 修改 Pump 状态为 `offline` 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 paused，现在不想使用这个 Pump 了，能否用 update-pump 命令设置状态为 offline？
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```

File: dev/reference/tidb-binlog/faq.md [hunk 8]
```diff
@@ -115,9 +115,96 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+
+## 可以使用 binlogctl 的 `update-pump`/`update-drainer` 命令来暂停 Pump/Drainer 服务吗？
+
+不可以。`update-pump`/`update-drainer` 命令直接修改 PD 中保存的状态信息。执行这个命令并不会通知 Pump/Drainer 做相应的操作，**而且使用不当会使数据同步中断，甚至造成数据丢失。**
+
+## 什么情况下使用 binlogctl 的 `update-pump` 命令设置 Pump 状态为 `paused`？
+
+在某些异常情况下，Pump 没有正确维护自己的状态，实际上状态应该为 `paused`。这时可以使用 `update-pump` 对状态进行修正，例如：
+
+- Pump 异常退出（可能由 panic 或者误操作执行 kill -9 直接 kill 掉进程导致），Pump 保存在 PD 中的状态仍然为 `online`。如果暂时不需要重启 Pump 恢复服务，可以使用 `update-pump` 把该 Pump 状态设置为 `paused`，避免对 TiDB 写 binlog 和 Drainer 获取 binlog 造成干扰。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
- Pump 异常退出（可能由 panic 或者误操作执行 `kill -9` 命令直接 kill 掉进程导致），Pump 保存在 PD 中的状态仍然为 `online`。如果暂时不需要重启 Pump 恢复服务，可以使用 `update-pump` 把该 Pump 状态设置为 `paused`，避免对 TiDB 写 binlog 和 Drainer 获取 binlog 造成干扰。
```

File: dev/reference/tidb-binlog/faq.md (hunk 6 above)
Comment (anotherrachel) [reply dev/reference/tidb-binlog/faq.md]: I agree.

File: dev/reference/tidb-binlog/faq.md [hunk 9]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+- 同步任务取消：不再需要将数据同步到某个下游，所以下线对应的 Drainer。
+- 服务器迁移：需要将服务迁移到其他服务器。下线服务，在新的服务器上重新部署。
+
+## 可以通过哪些方式暂停 Pump/Drainer？
+
+- 直接 kill 进程。
+
+    >**注意：**
+    >
+    > 不能使用 `kill -9`命令，否则 Pump/Drainer 无法对信号进行处理。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
    > 不能使用 `kill -9` 命令，否则 Pump/Drainer 无法对信号进行处理。
```

File: dev/reference/tidb-binlog/faq.md [hunk 10]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+> 仅在可以容忍 binlog **数据丢失、上下游数据不一致**或者确认不再需要使用该 Pump 存储的 binlog 数据的情况下，才能使用 `update-pump` 修改 Pump 状态为 `offline`。
+
+可以使用 `update-pump` 修改 Pump 状态为 `offline` 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
## Pump 进程已经退出，且状态为 `paused`，现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```

File: dev/reference/tidb-binlog/faq.md [hunk 11]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+
+Pump 以 `paused` 状态退出进程时，不保证所有 binlog 数据被下游 Drainer 消费。所以这样做会有上下游数据不一致的风险。正确的做法是重新启动 Pump，然后使用 `offline-pump` 下线该 Pump。
+
+## 什么情况下使用 binlogctl 的 update-drainer 命令设置 Drainer 状态为 offline？
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/faq.md]:
```suggestion
## 什么情况下使用 binlogctl 的 `update-drainer` 命令设置 Drainer 状态为 offline？
```

File: v2.1/reference/tidb-binlog/faq.md [hunk 12]
```diff
@@ -99,25 +99,114 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
+
+Pump 以 `paused` 状态退出进程时，不保证所有 binlog 数据被下游 Drainer 消费。所以这样做会有上下游数据不一致的风险。正确的做法是重新启动 Pump，然后使用 `offline-pump` 下线该 Pump。
+
+## 什么情况下使用 binlogctl 的 update-drainer 命令设置 Drainer 状态为 offline？
```
Comment (TomShawn) [inline v2.1/reference/tidb-binlog/faq.md]:
```suggestion
## 什么情况下使用 binlogctl 的 `update-drainer` 命令设置 Drainer 状态为 offline？
```

File: v2.1/reference/tidb-binlog/faq.md [hunk 13]
```diff
@@ -99,25 +99,114 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+> 仅在可以容忍 binlog **数据丢失、上下游数据不一致**或者确认不再需要使用该 Pump 存储的 binlog 数据的情况下，才能使用 `update-pump` 修改 Pump 状态为 `offline`。
+
+可以使用 `update-pump` 修改 Pump 状态为 `offline` 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```
Comment (TomShawn) [inline v2.1/reference/tidb-binlog/faq.md]:
```suggestion
## Pump 进程已经退出，且状态为 `paused`，现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```

File: v3.0/reference/tidb-binlog/faq.md [hunk 14]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+- 同步任务取消：不再需要将数据同步到某个下游，所以下线对应的 Drainer。
+- 服务器迁移：需要将服务迁移到其他服务器。下线服务，在新的服务器上重新部署。
+
+## 可以通过哪些方式暂停 Pump/Drainer？
+
+- 直接 kill 进程。
+
+    >**注意：**
+    >
+    > 不能使用 `kill -9`命令，否则 Pump/Drainer 无法对信号进行处理。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/faq.md]:
```suggestion
    > 不能使用 `kill -9` 命令，否则 Pump/Drainer 无法对信号进行处理。
```

File: v3.0/reference/tidb-binlog/faq.md [hunk 15]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
+
+Pump 以 `paused` 状态退出进程时，不保证所有 binlog 数据被下游 Drainer 消费。所以这样做会有上下游数据不一致的风险。正确的做法是重新启动 Pump，然后使用 `offline-pump` 下线该 Pump。
+
+## 什么情况下使用 binlogctl 的 update-drainer 命令设置 Drainer 状态为 offline？
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/faq.md]:
```suggestion
## 什么情况下使用 binlogctl 的 `update-drainer` 命令设置 Drainer 状态为 offline？
```

File: v3.0/reference/tidb-binlog/faq.md [hunk 16]
```diff
@@ -99,25 +99,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+> 仅在可以容忍 binlog **数据丢失、上下游数据不一致**或者确认不再需要使用该 Pump 存储的 binlog 数据的情况下，才能使用 `update-pump` 修改 Pump 状态为 `offline`。
+
+可以使用 `update-pump` 修改 Pump 状态为 `offline` 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/faq.md]:
```suggestion
## Pump 进程已经退出，且状态为 `paused`，现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```

File: v3.1/reference/tidb-binlog/faq.md [hunk 17]
```diff
@@ -98,25 +98,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+> 仅在可以容忍 binlog **数据丢失、上下游数据不一致**或者确认不再需要使用该 Pump 存储的 binlog 数据的情况下，才能使用 `update-pump` 修改 Pump 状态为 `offline`。
+
+可以使用 `update-pump` 修改 Pump 状态为 `offline` 的情况有：
+
+- 在某些情况下，Pump 异常退出进程，且无法恢复服务，同步就会中断。如果希望恢复同步且可以容忍部分 binlog 数据丢失，可以使用 `update-pump` 命令将该 Pump 状态设置为 `offline`，则 Drainer 会放弃拉取该 Pump 的 binlog 然后继续同步数据。
+- 有从历史任务遗留下来且不再使用的 Pump 且进程已经退出（例如测试使用的服务），之后不再需要使用该服务，使用 `update-pump` 将该 Pump 设置为 `offline`。
+
+在其他情况下一定要使用 `offline-pump` 命令让 Pump 走正常的下线处理流程。
+
+## Pump 进程已经退出，且状态为 `paused`。现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```
Comment (TomShawn) [inline v3.1/reference/tidb-binlog/faq.md]:
```suggestion
## Pump 进程已经退出，且状态为 `paused`，现在不想使用这个 Pump 节点了，能否用 binlogctl 的 `update-pump` 命令设置节点状态为 `offline`？
```

File: v3.1/reference/tidb-binlog/faq.md [hunk 18]
```diff
@@ -98,25 +98,112 @@ Drainer 启动的时候会去读取 checkpoint，如果读取不到，就会使
...
+
+Pump 以 `paused` 状态退出进程时，不保证所有 binlog 数据被下游 Drainer 消费。所以这样做会有上下游数据不一致的风险。正确的做法是重新启动 Pump，然后使用 `offline-pump` 下线该 Pump。
+
+## 什么情况下使用 binlogctl 的 update-drainer 命令设置 Drainer 状态为 offline？
```
Comment (TomShawn) [inline v3.1/reference/tidb-binlog/faq.md]:
```suggestion
## 什么情况下使用 binlogctl 的 `update-drainer` 命令设置 Drainer 状态为 offline？
```

**Pull request merged**